core/trino-main/src/main/java/io/trino/sql/planner/planprinter/JsonRenderer.java

```diff
@@ -59,15 +61,24 @@ private JsonRenderedNode renderJson(PlanRepresentation plan, NodeRepresentation
         private final String id;
```

Comment:
We're still missing `Estimates` in the json representation as compared to the text plan, can we add that in another commit ?